nixos/modules/services/web-apps/matomo.nix

```diff
@@ -2,15 +2,14 @@
 with lib;
 let
   cfg = config.services.matomo;
+  fpm = config.services.phpfpm.pools.${pool};
 
   user = "matomo";
   dataDir = "/var/lib/${user}";
   deprecatedDataDir = "/var/lib/piwik";
 
   pool = user;
-  # it's not possible to use /run/phpfpm/${pool}.sock because /run/phpfpm/ is root:root 0770,
-  # and therefore is not accessible by the web server.
-  phpSocket = "/run/phpfpm-${pool}.sock";
+  phpSocket = "/run/phpfpm/${pool}.sock";
```

Comment:
You can get rid of this and use `fpm.socket` instead, if I recall correctly.

Reply:
When I tried that there or in the place it's used I get this;
`error: attribute 'matomo' missing, at /home/kiwi/nixpkgs/nixos/modules/services/web-apps/matomo.nix:5:9`

Reply:
I don't understand. `phpSocket` is only referenced once throughout `matomo.nix` and it is in documentation. I'll have to checkout your PR and poke around a bit tonight.